### PR TITLE
Article 025: We all hate Bugs

### PR DESCRIPTION
**Pull request by Teifun2**
#### Length of text ####

 :book:  long 

####  Difficulty to read ####
 :mouse:  light 

####  Difficulty to understand #### 
 :school_satchel:  easy 

####  Type of text #### 
 :laughing:  funny

####  Type of header #### 
 

 :boom:  catchy

### CODE REVIEW

**Comment by Teifun2** [comment]
**Plenum:**
Problems:
- Not short!
- Not Informative!
- Title without question.

Ideas:
- Add Bug Explenation at the begining.
- When removing bugs you add another
- Hackers love Bugs (next Page)
- Can be using you

File: articles/article-025-we-all-hate-bugs.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+They hide in corners and behind large furniture and are most of the times hard to find. But whether in the household or in the software code, if you find a bug that was bothering you for a long time the feeling of success is rewarding.
+A bug in software is usually a behavior of the software which is not intended. Sometimes they are very hard to find and can be attributed to one single character that was wrongly placed. Other times they are structural mistakes that lead to a "bridge collapse". The act of finding bugs is called Debugging. Like pest control, the code is searched and cleaned from every bug that would influence the behavior of the software in a negative way.
+Often while fixing software bugs the programmer accidentally adds another bug. The cicle is endless.
+<!-- END BODY -->
+
+![Bugs](../images/image-025-we-all-hate-bugs.jpeg)
+
+## Optional
+<!-- BEGIN OPTIONAL -->
+Some People say they spend more then 50% of their work with debugging.
```
Comment (s3lph) [inline articles/article-025-we-all-hate-bugs.md]: `s/then/than/`

**Comment by MoiraZuber** [comment]
Added image with no significant changes.

**Comment by Teifun2** [comment]
> Not short!
> Not Informative!
> Title without question.
> When removing bugs you add another
> Can be using you

Implemented all these changes.

> Add Bug Explenation at the begining.

I would like to keep the current ordering because it makes a better narrative in my opinion. (please advise)

> s/then/than/

fixed that

File: articles/article-025-we-all-hate-bugs.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!-- BEGIN TITLE -->
+# We all hate Bugs
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Bugs are something nobody wants. They creep into your house or your code without notice. There they eat your resources and clog your sink.
```
Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: Do we need the first sentence? Its already reflected in the title.

Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: I would not refer to code in this sentence. The concept of a software bug is introduce only later.

File: articles/article-025-we-all-hate-bugs.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<!-- BEGIN TITLE -->
+# We all hate Bugs
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Bugs are something nobody wants. They creep into your house or your code without notice. There they eat your resources and clog your sink. 
+They hide in corners and behind large furniture and are most of the times hard to find. But whether in the household or in the software code, if you find a bug that was bothering you for a long time the feeling of success is rewarding.
+A bug in software is usually a behavior of the software which is not intended. Sometimes they are very hard to find and can be attributed to one single character that was wrongly placed. Other times they are structural mistakes that lead to a "bridge collapse". The act of finding bugs is called Debugging. Like pest control, the code is searched and cleaned from every bug that would influence the behavior of the software in a negative way.
```
Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: Language: Sometimes they are very hard to find -> It is often very hard to find

Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: The bridge collapse is difficult to understand. But the article would work without the distinction between the two types of bugs (single character / structural). Also, is a structural bug really a bug or just a design flaw?

Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: that would influence -> that influences?
in a negative way -> negatively?

File: articles/article-025-we-all-hate-bugs.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Often while fixing software bugs the programmer accidentally adds another bug. The cicle is endless.
```
Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: Suggestion: Unfortunately, while fixing software bugs, ...

**Comment by Teifun2** [comment]
I tried to implement all suggestions.

File: articles/article-025-we-all-hate-bugs.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<!-- BEGIN TITLE -->
+# We all hate Bugs
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Bugs creep into your house without notice, there they eat your food and clog your sink. 
+They hide in corners and behind large furniture and are most of the times hard to find. If you find a bug that was bothering you for a long time the feeling of success is rewarding.
+A bug in software is very simmilar. It's usually a behavior of the software which is not intended. It is often very hard to find bugs in software and can sometimes be attributed to one single character that was wrongly placed. The act of finding bugs is called Debugging. Like pest control, the code is searched and cleaned from every bug that influences the behavior negatively.
+Unfortunately, while fixing software bugs the programmer accidentally adds other bugs. The cicle is endless.
```
Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: cicle -> Cycle

File: articles/article-025-we-all-hate-bugs.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+<!-- BEGIN TITLE -->
+# We all hate Bugs
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Bugs creep into your house without notice, there they eat your food and clog your sink. 
+They hide in corners and behind large furniture and are most of the times hard to find. If you find a bug that was bothering you for a long time the feeling of success is rewarding.
+A bug in software is very simmilar. It's usually a behavior of the software which is not intended. It is often very hard to find bugs in software and can sometimes be attributed to one single character that was wrongly placed. The act of finding bugs is called Debugging. Like pest control, the code is searched and cleaned from every bug that influences the behavior negatively.
```
Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: It's usually -> it is a  ...
or could bugs be other things than behavior, which is not intended?

Comment (marcelluethi) [inline articles/article-025-we-all-hate-bugs.md]: Debugging->debugging

**Comment by OmniaKahla** [comment]
typos have been resolved.

**Comment by marcelluethi** [comment]
@Teifun2 Just noticed that in my last review I forgot to change the label to revise

**Comment by Teifun2** [comment]
I fixed all the typos. For sake of speed i would just proceed to testing, but if you think it needs to go to plenum thats ok too.

**Comment by marcelluethi** [comment]
@Teifun2  I found a few more language issues (commas, typos). Once they are corrected, you can just put it to testing. No need to wait for another review.

**Pull request merged**